common/ideas/00_basic_ideas.txt

```diff
@@ -705,350 +673,369 @@ maritime_ideas = {
+		discipline = 0.05
 	}
 
 	ai_will_do = {
-		factor = 0.9
-		modifier = {
-			factor = 0
-			has_idea_group = religious_ideas
-		}
+		factor = 0.94
 	}
 }
 
-influence_ideas = {
-	category = DIP
+
+defensive_ideas = {
+	category = MIL
 
 	bonus = {
-		unjustified_demands = -0.5
+		hostile_attrition = 3
 	}
 
-	tribute_system = {
-		vassal_income = 0.25
+	battlefield_commisions = {
+		army_tradition_from_battle = 1.0
 	}
-
-	additional_loyalist_recruitment = {
-		reduced_liberty_desire = 15
+	military_drill = {
+		land_morale = 0.15
 	}
-
-	integrated_elites = {
-		diplomatic_annexation_cost = -0.25
+	improved_manuever = {
+		leader_land_manuever = 1
 	}
-	buffer_states = {
-		diplomatic_upkeep = 1
+	regimental_system = {
+		land_maintenance_modifier = -0.05
 	}
-	diplomatic_influence = {
-		diplomatic_reputation = 2
+	defensive_mentality = {
+		defensiveness = 0.2
+		fort_maintenance_modifier = -0.25
 	}
-	postal_service = {
-		envoy_travel_time = -0.25
+	supply_trains = {
+		reinforce_speed = 0.33
 	}
-	marcher_lords = {
-		vassal_forcelimit_bonus = 1
+	improved_foraging = {
+		land_attrition = -0.33
 	}
 
 	ai_will_do = {
-		factor = 0.5
-		modifier = {
-			factor = 1.5
-			is_emperor = yes
-		}
-		modifier = {
-			factor = 1.7
-			vassal = 1
-		}
-		modifier = {
-			factor = 1.7
-			vassal = 2
-		}
+		factor = 0.77
 	}
 }
 
-naval_ideas = {
-	category = MIL
 
-	trigger = {
-		primitives = no
-	}
+quality_ideas = {
+	category = MIL
 
 	bonus = {
-		ship_durability = 0.1
-		allowed_marine_fraction = 0.05
+		discipline = 0.05
 	}
 
-	boarding_parties = {
-		leader_naval_shock = 1
+	private_to_marshal = {
+		infantry_power = 0.1
 	}
-	improved_rams = {
-		galley_power = 0.25
+	quality_education = {
+		army_tradition = 0.5
 	}
-	naval_cadets = {
-		leader_naval_fire = 1
-		sunk_ship_morale_hit_recieved = -0.33
+	finest_of_horses = {
+		cavalry_power = 0.1
 	}
-	naval_glory = {
-		navy_tradition = 1
+	escort_ships = {
+		ship_durability = 0.05
 	}
-	press_gangs = {
-		sailors_recovery_speed = 0.25
+	naval_drill = {
+		naval_morale = 0.10
 	}
-	oak_forests_for_ships = {
-		heavy_ship_power = 0.20
+	copper_bottoms = {
+		naval_attrition = -0.25
 	}
-	superior_seamanship = {
-		naval_morale = 0.1
-		global_naval_engagement_modifier = 0.1
+	massed_battery = {
+		artillery_power = 0.1
 	}
 
 	ai_will_do = {
-		factor = 0.31
-		modifier = {
-			factor = 0
-			is_tribal = yes
-		}
-		modifier = {
-			factor = 0
-			primitives = yes
-		}
-		modifier = {
-			factor = 0
-			NOT = { num_of_ports = 1 }
-			NOT = { is_colonial_nation = yes }
-		}
-		modifier = {
-			factor = 0.31
-			NOT = { num_of_ports = 10 }
-		}
-		modifier = {
-			factor = 0.31
-			num_of_cities = 50
-		}
+		factor = 0.49
+	}
+}
 
-		modifier = {
-			factor = 10
-			OR = {
-				has_reform = pirate_republic_reform
-				has_reform = pirate_kingdom
-				has_reform = war_against_the_world_government
-				has_reform = black_market_consortium_government
-			}
-		}
+
+quantity_ideas = {
```

Comment:
It could perhaps be interesting to slightly tone down the amount of manpower/FL quantity gives, and instead either give a bit of extra manpower to other ideas or grant them some other way of de-facto having more sustainability (cheaper leaders for more slacken, mercs cost less professionalism to hire, yearly professionalism gain, scaling manpower modifier with some condition, like amount of subjects, colonial nations, fleet size or true faith religion). This is just so quantity isn't always a must-pick due to it being the only source of extra manpower/FL aside from aristocratic